examples/material_gallery/flutter.yaml

```diff
@@ -7,6 +7,16 @@ assets:
   - assets/section_components.png
   - assets/section_patterns.png
   - assets/section_usability.png
+  - packages/flutter_gallery_assets/sun.png
```

Comment:
All made from scratch, except the snow flakes which are public domain from Wikipedia (marked as free for commercial use with modification without attribution).